index.html

```diff
@@ -26,14 +26,16 @@ <h2>Definitions</h2>
         <dfn>display-surface</dfn> and <dfn>browser</dfn> [=display-surface=].
       </p>
       <p>
-        Recall that when {{MediaDevices/getDisplayMedia()}} is called and the user chooses to share
-        some [=display-surface=], exactly one video track is produced. If the user chooses to
-        capture a [=browser=] [=display-surface=] associated with the [=browsing context=] from
-        which {{MediaDevices/getDisplayMedia()}} was called, we define the resulting capture session
-        as <dfn>self-capture</dfn>, and this video track a <dfn>self-capture video track</dfn>.
+        Let <var>MD</var> be a {{MediaDevices}} object. Let <var>AD</var> be the [=associated
+        Document=] of <var>MD</var>'s [=relevant global object=]. Let <var>TLBC</var> be
+        <var>AD</var>'s [=top-level browsing context=]. Assume
+        <var>MD</var>.{{MediaDevices/getDisplayMedia()}} is called. If the user chooses to capture a
+        [=browser=] [=display-surface=] associated with <var>TLBC</var>, we define the resulting
+        capture session as <dfn>self-capture</dfn>. The call results in exactly one video track,
+        which we define to be a <dfn>self-capture video track</dfn>.
```

Comment:
"Let" is for algorithms, and this is really be the "§ Terminology" section, where we should express concepts as straightforwardly as possible, especially this early in the document. If we feel we need algorithms, we could reference and define them elsewhere. But I'd like to take another stab first.

To tweak some more, "user chooses" implies getDisplayMedia, and we want a definition that also works with e.g. getViewportMedia. I think we can define self-capture regardless of how capture is obtained:
```suggestion
        This specification defines <dfn>self-capture</dfn> as the capture of a
        [=browser=] [=display-surface=] that is the rendered form of the
        [=top-level browsing context=] of the [=associated Document=] of the
        {{MediaDevices}} object from which the application initiated the
        capture session. A <dfn>self-capture video track</dfn> is a
        {{MediaStreamTrack}} sourced by <a>self-capture</a>.
```

Reply:
Hard to read, but we can iterate later - let's go with this for now.